notebooks/DAAC_UWG_cloud_tutorial/nsidc_daac_uwg_cloud_access_tutorial.ipynb

```diff
@@ -0,0 +1,430 @@
+    "|:-----------:|:---------------------|\n",
+    "| ATL03 | ATLAS/ICESat-2 L2A Global Geolocated Photon Data |\n",
+    "| ATL06 | ATLAS/ICESat-2 L3A Land Ice Height |\n",
+    "| ATL07 | ATLAS/ICESat-2 L3A Sea Ice Height |\n",
+    "| ATL08 | ATLAS/ICESat-2 L3A Land and Vegetation Height |\n",
+    "| ATL09 | ATLAS/ICESat-2 L3A Calibrated Backscatter Profiles and Atmospheric Layer Characteristics |\n",
+    "| ATL10 | ATLAS/ICESat-2 L3A Sea Ice Freeboard |\n",
+    "| ATL11 | ATLAS/ICESat-2 L3B Slope-Corrected Land Ice Height Time Series |\n",
+    "| ATL12 | ATLAS/ICESat-2 L3A Ocean Surface Height |\n",
+    "| ATL13 | ATLAS/ICESat-2 L3A Along Track Inland Surface Water Data |"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57fb1273",
+   "metadata": {},
+   "source": [
+    "### Search for cloud-hosted data\n",
+    "For most collections, to search for only data in the cloud, the `cloud_hosted` method can be used. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25405fb6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Query = earthaccess.collection_query().keyword('ICESat-2').cloud_hosted(True)\n",
+    "print(Query.hits())\n",
+    "collections = Query.fields(['ShortName', 'Version']).get(10)\n",
+    "print(collections)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "292d8c2d",
+   "metadata": {},
+   "source": [
+    "## Search a data set using spatial and temporal filters \n",
+    "\n",
+    "As an example of a search using spatial and temporal filters, we'll search for ATL06 granules over the Juneau Icefield, AK, for March 2020.\n",
+    "\n",
+    "The ATL06 version 005 collection is identified by the `concept_id` C2153572614-NSIDC_CPRD.  The temporal range is identified with standard date strings, and latitude-longitude corners of a bounding box is specified.  Polygons and points, as well as shapefiles can also be specified."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3e8b65d1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Query = earthaccess.granule_query().concept_id(\n",
+    "    'C2153572614-NSIDC_CPRD'\n",
+    ").temporal(\n",
+    "    \"2020-03-01\", \"2020-03-30\"\n",
+    ").bounding_box(\n",
+    "    -134.7,58.9,-133.9,59.2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc6924e3",
+   "metadata": {},
+   "source": [
+    "As before, we can use `Query.hits()` to find the number of granules that match our search. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "15163612",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Query.hits()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1f65f1e6",
+   "metadata": {},
+   "source": [
+    "We'll get metadata for these 4 granules and display it.  The rendered metadata shows a download link, granule size and two images of the data.  _This doesn't render so well on this Jupyter Hub._\n",
```

Comment:
Not sure if we want to leave this comment in about it not rendering so well?

Reply:
Agreed, will remove

Reply:
Agree.